costmap_2d/src/layered_costmap.cpp

```diff
@@ -174,6 +200,21 @@ bool LayeredCostmap::isCurrent()
   return current_;
 }
 
+costmap_2d::Costmap2D* LayeredCostmap::getCostmap(double t)
+{
+  if (timed_costmaps_.empty())
+    return NULL;
```

Comment:
```suggestion
    return nullptr;
```
F: use `nullptr` in modern cpp